src/Controls/src/Core/TitleBar/TitleBar.cs

```diff
@@ -266,6 +266,18 @@ private void TitleBar_PropertyChanged(object? sender, System.ComponentModel.Prop
 				Window.Activated += Window_Activated;
 				Window.Deactivated += Window_Deactivated;
 			}
+			else if (e.PropertyName == nameof(LeadingContent))
```

Comment:
Can these all be moved into the OnPropertyChanged delegates for these BPs?

For example, `OnLeadingChanged`